README.md

```diff
@@ -476,7 +476,9 @@ import { h, Component, render } from 'preact';
 // Enable debug mode. You can reduce the size of your app by only including this
 // module in development builds. eg. In Webpack, wrap this with an `if (module.hot) {...}`
 // check.
-require('preact/debug');
+if (process.env.NODE_ENV !== 'production') {
```

Comment:
if webpack is set  correctly this check should be sufficient, no need to wrap with `module.hot` check